.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Expected behavior
+
+## Actual behavior
+
+## Reproducing the behavior
+
+### Computer that is creating the backups
+
+* Operating system, e.g. `Ubuntu Server 14.04.5`:
+* Ruby version, e.g. `2.3.1`:
+* Version of backup gem, e.g. `4.3.0`:
+* Versions of tools or services, e.g. `MySQL 5.7.9`:
+
+### Computer that is restoring the backups
+
+* Operating system, e.g. `Ubuntu Server 14.04.5`:
+* Ruby version, e.g. `2.3.1`:
+* Version of backup gem, e.g. `4.3.0`:
+* Versions of tools or services, e.g. `OpenSSL 0.9.8zh`:
+
+### Steps to reproduce the behavior
+
+1.
+2.
+3.
```

Comment:
Maybe we can ask people to provide their `config.rb` and related models as well.
We could even ask for a small project with a reproducible example.